etl/steps/data/garden/emissions/2023-11-23/national_contributions.meta.yml

```diff
@@ -1,330 +1,323 @@
         unit: tonnes
         short_unit: t
-        description: |
-          Cumulative CO2 emissions.
+        description_short: *measured-in-tonnes
       cumulative_emissions_n2o_fossil:
         title: Cumulative nitrous oxide emissions from fossil fuels and industry
         unit: tonnes
         short_unit: t
-        description: |
-          Cumulative nitrous oxide emissions from fossil fuels and industry.
+        description_short: *measured-in-co2-eq
       cumulative_emissions_n2o_land:
         title: Cumulative nitrous oxide emissions from agriculture and land use
         unit: tonnes
         short_unit: t
-        description: |
-          Cumulative nitrous oxide emissions from agriculture and land use.
+        description_short: *measured-in-co2-eq
       cumulative_emissions_n2o_total:
         title: Cumulative nitrous oxide emissions
         unit: tonnes
         short_unit: t
-        description: |
-          Cumulative nitrous oxide emissions.
+        description_short: *measured-in-co2-eq
+      # Temperature response to emissions of CH4, CO2, N2O and GHG, in °C (as originally given in the data).
       temperature_response_ghg_fossil:
         title: Change in global mean surface temperature caused by greenhouse gas emissions from fossil fuels and industry
         unit: °C
         short_unit: °C
-        description: |
-          Change in global mean surface temperature (in °C) caused by greenhouse gas emissions from fossil fuels and industry.
+        description_short: *measured-in-celsius
       temperature_response_ghg_land:
         title: Change in global mean surface temperature caused by greenhouse gas emissions from agriculture and land use
         unit: °C
         short_unit: °C
-        description: |
-          Change in global mean surface temperature (in °C) caused by greenhouse gas emissions from agriculture and land use.
+        description_short: *measured-in-celsius
       temperature_response_ghg_total:
         title: Change in global mean surface temperature caused by greenhouse gas emissions
         unit: °C
         short_unit: °C
-        description: |
-          Change in global mean surface temperature (in °C) caused by greenhouse gas emissions. This measures each country's contribution to global mean surface temperature (GMST) rise from its cumulative emissions of carbon dioxide, methane and nitrous oxide. The warming effects of each gas are calculated based on cumulative CO2-equivalent emissions using the Global Warming Potential (GWP*) approach.
+        description_short: *measured-in-celsius
+        description_key:
+          - This temperature change measures each country's contribution to global mean surface temperature (GMST) rise from its cumulative emissions of carbon dioxide, methane and nitrous oxide.
+          - The warming effects of each gas are calculated based on cumulative CO₂-equivalent emissions using the Global Warming Potential (GWP*) approach.
```

Comment:
do you describe what the Global Warming Potential approach means anywhere? Feels like it could be useful.

Reply:
Good idea, I'll keep that in mind when improving the metadata of all co2 charts on the topic page.